helpdesk/query.py

```diff
@@ -162,13 +165,18 @@ def get_datatables_context(self, **kwargs):
         to a Serializer called DatatablesTicketSerializer in serializers.py.
         """
         objects = self.get()
-        order_by = "-created"
         draw = int(kwargs.get("draw", [0])[0])
         length = int(kwargs.get("length", [25])[0])
         start = int(kwargs.get("start", [0])[0])
         search_value = kwargs.get("search[value]", [""])[0]
-        order_column = kwargs.get("order[0][column]", ["5"])[0]
-        order = kwargs.get("order[0][dir]", ["asc"])[0]
+
+        sorting = self.params.get("sorting", "created")
+        default_order_col = DATATABLES_COLUMN_NUM_LOOKUP.get(sorting, "5")
+        sortreverse = self.params.get("sortreverse", None)
+        default_order = "desc" if sortreverse else "asc"
+
+        order_column = kwargs.get("order[0][column]", [default_order_col])[0]
```

Comment:
This will never use the `default_order_col` because the DataTable defaults to the `id` field.